src/main/scala/com/gu/contentapi/utils/PredicateUtils.scala

```diff
@@ -0,0 +1,20 @@
+package com.gu.contentapi.utils
+
+object PredicateUtils {
+
+  implicit class RichPredicate[A](val f: Function1[A, Boolean]) extends AnyVal {
+    def apply(v: A) = f(v)
+
+    def &&(g: Function1[A, Boolean]): Function1[A, Boolean] = {
```

Comment:
@mchv in case you missed my previous comment, any reason I haven't seen why we're not simply using another `filter`?